lib/model-builder.js

```diff
@@ -426,6 +426,9 @@ ModelBuilder.prototype.define = function defineClass(className, properties, sett
       var requiredOptions = typeof prop.required === 'object' ? prop.required : undefined;
       ModelClass.validatesPresenceOf(propertyName, requiredOptions);
     }
+    if (DataType === Date) {
```

Comment:
Can we do a one liner if statement here? `if (DataType === Date) ModelClass.validatesDateOf(propertyName);`